src/js/claims-status/containers/ClaimsStatusApp.jsx

```diff
@@ -28,9 +29,11 @@ class ClaimsStatusApp extends React.Component {
         userProfile={this.props.profile}
         loginUrl={this.props.loginUrl}
         verifyUrl={this.props.verifyUrl}>
-        <AppContent>
-          {this.props.children}
-        </AppContent>
+        <DowntimeNotification appTitle="claims status tool" dependencies={[services.evss]}>
```

Comment:
The former was my thinking too, but they'll actually be asked to login before they land on the application anyhow because of the way the menu links works with the `?next=` redirect. We have some work planned for a third piece that will tell the user in the login window prior to logging in if the application is experiencing downtime but for now it only checks when landing on the application. 

If they navigate directly (via bookmark or something) order would matter though - do you think it's worth making the change?

Reply:
Originally we were planning on difference messaging based on authenticated or not authenticated, so the downtime notifications had to wait anyway.

Reply:
Ahh, yeah that makes sense. I wouldn't worry about it.

At first, I didn't think the redirects happened until after the page was loaded up, but then I found this little number in `src/js/login/containers/Main.jsx`

```js
  bindNavbarLinks() {
    [...document.querySelectorAll('.login-required')].forEach(el => {
      el.addEventListener('click', e => {
        e.preventDefault();
        const nextQuery = { next: el.getAttribute('href') };
        const nextPath = appendQuery('/', nextQuery);
        history.pushState({}, el.textContent, nextPath);
        this.props.toggleLoginModal(true);
      });
    });
  }
```

I don't like :point_up: because it's not obvious that this is happening, but it's there and out of scope for this PR, so...I would say don't worry about it. The way you've got it right now makes enough sense. Navigating directly to an app with a required login is probably not super common (though I would love to see analytics on that).

Reply:
Yeah exactly! I definitely agree. That's something we might be working with later on.